src/widgets/geo-search/GeoSearchRenderer.js

```diff
@@ -20,7 +20,7 @@ const refineWithMap = ({ refine, paddingBoundingBox, mapInstance }) => {
     .getProjection()
     .fromLatLngToPoint(mapInstance.getBounds().getSouthWest());
 
-  southWestPoint.x = southWestPoint.x + paddingBoundingBox.right / scale;
+  southWestPoint.x = southWestPoint.x + paddingBoundingBox.left / scale;
```

Comment:
Nice catch!

Reply:
That's the story of my first CS project at university.